core-api/src/main/java/com/optimizely/ab/bucketing/Bucketer.java

```diff
@@ -123,42 +128,56 @@ private Variation bucketToVariation(@Nonnull Experiment experiment,
      *
      * @param experiment  The Experiment in which the user is to be bucketed.
      * @param bucketingId string A customer-assigned value used to create the key for the murmur hash.
+     * @param projectConfig      The current projectConfig
+     * @param options            An array of decision options
+     * @param reasons            Decision log messages
      * @return Variation the user is bucketed into or null.
      */
     @Nullable
     public Variation bucket(@Nonnull Experiment experiment,
                             @Nonnull String bucketingId,
-                            @Nonnull ProjectConfig projectConfig) {
+                            @Nonnull ProjectConfig projectConfig,
+                            @Nonnull List<OptimizelyDecideOption> options,
```

Comment:
am not sure, if we need options in bucketing logic.

Reply:
Not used for now, but will keep this consistently for future options.